src/common/IOSEnvironmentSetup.ts

```diff
@@ -142,7 +142,10 @@ export class IOSEnvironmentSetup extends BaseSetup {
             } else {
                 return new Promise<string>((resolve, reject) =>
                     reject(
-                        util.format(this.unfulfilledMessage, supportedRuntimes)
+                        util.format(
+                            this.unfulfilledMessage,
+                            `${iOSConfig.supportedRuntimes}`
```

Comment:
This was supposed to be showing the runtimes supported by the plugin. Instead it was showing the empty list of matching runtimes, in this case.